cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -47,7 +47,7 @@ func CreateInitStaticPodManifestFiles(manifestDir string, cfg *kubeadmapi.InitCo
 
 // GetStaticPodSpecs returns all staticPodSpecs actualized to the context of the current configuration
 // NB. this methods holds the information about how kubeadm creates static pod manifests.
-func GetStaticPodSpecs(cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.APIEndpoint, k8sVersion *version.Version) map[string]v1.Pod {
+func GetStaticPodSpecs(cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.APIEndpoint) map[string]v1.Pod {
```

Comment:
can you move this refactoring to a separate PR ?

Reply:
ok, @SataQiu 
please move :
> - Fix the bug that k8sVersion is unused in kubeadm/app/phases/controlplane/manifests.go#getControllerManagerCommand function.
> - manifests.go#GetStaticPodSpecs uses cfg.KubernetesVersion instead of k8sVersion to generate manifests. This PR cleans up redundant k8sVersion parameter.

to separate PR.

Reply:
ideally in other projects i'd just recommend a separate commit, but let's follow what k/k does / recommends.